rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -0,0 +1,233 @@
+#ifdef QT_UI
```

Comment:
Welp, this is a nice question. Which coding style I should use in rpcs3qt? Qt one or RPCS3?
Currently I use Qt, but I can change it to default RPCS3 coding style while there not to much code done.